src/Components/Loader/Loader.js

```diff
@@ -0,0 +1,49 @@
+import React, { useState, useEffect } from "react";
+import PropTypes from "prop-types";
+import styled from "styled-components";
+import loaderImage from "./instagram.gif";
+
+const LoaderContainer = styled.div`
+  position: fixed;
+  top: 0;
+  left: 0;
+  z-index: 10000;
+  background: #fff;
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  height: 100%;
+  width: 100%;
+`;
+
+const LoaderImage = styled.img`
+  max-width: 100%;
+  height: auto;
+`;
+
+function Loader({ fadeOutInterval }) {
+  const [fadeOut, setFadeOut] = useState(false);
+
+  useEffect(() => {
+    const timeout = setTimeout(() => {
```

Comment:
I think you should add an if statement to check if the component will be timed. 
For example, what if I want to show the component until the data is fetched? No one knows how long it will take to fetch the data.
So if you check if the component will be used with timing or not, you can use it in two scenarios:
- If you want to display the component for a specific amount of time, which applies in your case to App.js right now because you want to show it maybe for 5 seconds on startup (Although 5 seconds is too much)
- The second scenario if you want to fetch data which means you don't know for how long you are going to wait.